template/src/components/DebugMenu.js

```diff
@@ -0,0 +1,259 @@
+import BottomSheet, {BottomSheetFlatList} from '@gorhom/bottom-sheet'
+import React, {useCallback, useMemo, useRef, useState} from 'react'
+import {
+  Modal,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  View,
+  Text,
+  Button,
+  StyleSheet,
+  useWindowDimensions,
+} from 'react-native'
+import {getApplicationName, getBuildNumber, getDeviceId} from 'react-native-device-info'
+import Draggable from 'react-native-draggable'
+
+import Config, {BOTTOM_SHEET_TYPE, CODEPUSH_KEYS, EXTRA_QA_ENVS} from '../constants/configs'
+
+import {InfoMenu, InfoMenuRow, InfoMenuLink} from './InfoMenu'
+
+import {colors, fonts, metrics} from '../themes'
+
+import {localize} from '../locale/I18nConfig'
+import {useDispatch, useSelector} from 'react-redux'
+import {getApiUrl, getCodePushKey} from '../store/selectors'
+import {appActions} from '../store/reducers'
+
+const debugMenuSize = 50
+
+const AppInfoSection = () => {
+  const appName = useMemo(() => getApplicationName(), [])
+  const buildNumber = useMemo(() => getBuildNumber(), [])
+  const deviceId = useMemo(() => getDeviceId(), [])
```

Comment:
Same above, move to `constants`

Reply:
why we need to move to constants
these are not a magic number

Reply:
But they will not change though all app life, so I think we can just move it to `constants` so we can use it somewhere else (if needed) without having to calls useMemo for this again

Reply:
where using, can import directly {getApplicationName, getBuildNumber, getDeviceId} from 'device-info'.
useMemo for render once, it's clear and not effect to the performance.

Reply:
Then we can skip this for now